src/js/components/organisms/page-footer/page-footer.vue.html

```diff
@@ -0,0 +1,27 @@
+<footer class="footer">
+    <div class="footer__container">
+        <div class="footer__primary">
+            <router-link to="/" class="branded-logo footer__branded-logo" aria-label="Ultimaker homepage">
+                <icon class="branded-logo__u" icon-name="ultimaker-u" />
+            </router-link>
+            <nav class="footer__navigation" v-if="navigation && navigation.length">
+                <footer-navigation-category :key="index" v-for="(navigationItem, index) in navigation" v-bind="navigationItem" />       
+            </nav>
+        </div>
+
+        <div class="footer__secondary">
+            <social-nav></social-nav>
+            <locale-selector class="footer__locale-selector"
+                            v-bind="localeSelector"
+                            :country="country"
+                            v-on="$listeners">
+            </locale-selector>
+        </div>
+
+        <div class="footer__tertiary">
+            <nav class="footer__legal-navigation" v-if="legalNavigation && legalNavigation.length">
+                <footer-navigation-category class="footer__legal-navigation__column" :label="copyrightLabel" :items="legalNavigation" />       
+            </nav>
+       </div>
+    </div>
+</footer>
```

Comment:
Your IDE/editor is lacking an EditorConfig plugin. There is a newline missing at the end of this file.